File: message-api-schema-validator/src/main/java/uk/ac/jisc/rdss/tests/vocabulary/read/VocabularyReadRequestTest.java
package uk.ac.jisc.rdss.tests.vocabulary.read;

import org.junit.Test;

import uk.ac.jisc.rdss.tests.AbstractSchemaValidatorTest;

public class VocabularyReadRequestTest extends AbstractSchemaValidatorTest {

    @Override
    protected String getSchemaFileName() {
        return "messages/vocabulary/read/request_schema.json";
    }

    @Test
    public void validateRequest() throws Exception {
        validateJson("messages/vocabulary/read/request.json");
    }
}

File: message-api-schema-validator/tests/vocabulary/vocabulary_read_request_test.py
from tests.abstract_schema_validator_test import AbstractSchemaValidatorTest
from unittest import TestCase


class VocabularyReadRequestTest(AbstractSchemaValidatorTest, TestCase):
    def get_json_schema_file_name(self):
        return 'messages/body/vocabulary/read/request_schema.json'

    def runTest(self):
        self.validate_json('messages/body/vocabulary/read/request.json')
